addons/mail/static/src/thread/message.xml

```diff
@@ -14,7 +14,7 @@
                 <span class="o-mail-own-name">
                     <strong class="o_redirect me-1 text-truncate cursor-pointer">
                     <t t-esc="author.name"/>
-                    </strong> - 
+                    </strong> -
                 </span>
```

Comment:
Out of scope of PR, but just commenting to remind changing that:
can clean this part of DOM that way:

```xml
<div class="o-mail-msg-header d-flex flex-wrap align-items-baseline">
    <strong class="o-mail-own-name o_redirect me-1 text-truncate cursor-pointer">
        <t t-esc="author.name"/>
     </strong>
     <span>-</span>
```

So that `.o-mail-own-name`.textNode() is properly just the author name, excluding `-`.